.github/workflows/pr-validate-changesets.yaml

```diff
@@ -6,6 +6,10 @@ on:
       - opened
       - edited
       - synchronize
+  workflow_run:
+    workflows: ["Tests"]
+    types:
+      - completed
```

Comment:
This doesn't seem necessary anymore as the changeset commit push from `fuel-service-user` will cancel the initial runs.

Reply:
Good catch, I had originally theorized we'd only want to validate the changesets once the tests have run but this will just prolong the workflow runs now.